instrumentation/github.com/aws/aws-lambda-go/otellambda/wrapHandler.go

```diff
@@ -38,7 +39,8 @@ func (h wrappedHandler) Invoke(ctx context.Context, payload []byte) ([]byte, err
 	defer h.instrumentor.tracingEnd(ctx, span)
 
 	if len(payload) > 0 {
-		span.SetAttributes(attribute.KeyValue{Key: "faas.event", Value: attribute.StringValue(string(payload))})
+		faasEventAttribute := obfuscator.ObfuscateAttributeValue(attribute.KeyValue{Key: "faas.event", Value: attribute.StringValue(string(payload))})
```

Comment:
Shouldn't you check if obfuscation is enabled?

Reply:
I'm checking inside ObfuscateAttributeValue since there I have the config, here I don't have them